infra/frontend/app-config/env-config/environment_variables.tf

```diff
@@ -9,6 +9,17 @@ locals {
     NODE_OPTIONS = "-r newrelic"
     # expose the current AWS Env to the FE Next Node Server at Runtime
     ENVIRONMENT = var.environment
+    # https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#labels
+    NEW_RELIC_LABELS = "app_name:${var.app_name};environment:${var.environment};service_name:${var.app_name}-${var.environment};serviceName:${var.app_name}-${var.environment};service.name:${var.app_name}-${var.environment};entity.name:${var.app_name}-${var.environment}"
```

Comment:
I added so many variants of `${var.app_name}-${var.environment}` because that's what APM looks for, and (in most situations) there's no harm in covering every base possible.